src/server/auth/server/testing/auth_test.go

```diff
@@ -1873,7 +1873,6 @@ func TestListDatum(t *testing.T) {
 		t.Skip("Skipping integration tests in short mode")
 	}
 	deleteAll(t)
-	defer deleteAll(t)
```

Comment:
What was the reason for removing this?

Reply:
Ah, yes, the test was failing but since it cleaned up after itself, I couldn't inspect the cluster state.  I see the problem here, though, which is that the auth credentials can't be accessed outside of the test and then you can't clean up your own cluster.  I'll revert this.